app-modules/prospect/src/Filament/Widgets/ProspectStats.php

```diff
@@ -52,28 +55,45 @@ protected function getStats(): array
         /** @var User $user */
         $user = auth()->user();
 
+        $alertStatusIds = AlertStatus::pluck('id')->toArray();
+
+        $alertQueryParams = [
+            'tableFilters' => [
+                'alerts' => [
+                    'values' => array_values($alertStatusIds),
+                ],
+                'subscribed' => [
+                    'isActive' => 'true',
+                ],
+            ],
+        ];
```

Comment:
Let's move this to be inline where it is used rather than define it way up here.